DogTales/dog/player/player.cpp

```diff
@@ -1,19 +1,24 @@
-#include <dog/player.hpp>
+#include <dog/player/player.hpp>
 
 namespace dog {
 Player::Player(bave::App& app, glm::vec2 const world_space) : m_app(app), m_world_space(world_space) {
 	m_sprite.set_size(size_v);
+
+	m_player_controller.bind_throttle("move_x", {.lo = bave::Key::eLeft, .hi = bave::Key::eRight});
```

Comment:
what will happen if both "Left" and "Right" are pressed simultaneously?

Reply:
"Left" will contribute `-1.0f` to the throttle, "Right" will contribute `+1.0f`, and they'll cancel each other out.

As you can imagine, this is a subjective/design level decision, rather than being "more technically correct" over any other approach.